sources/packages/backend/libs/services/src/queue/queue.service.ts

```diff
@@ -37,4 +42,32 @@ export class QueueService {
       dashboardReadonly: queue.queueConfiguration.dashboardReadonly,
     }));
   }
+
+  /**
+   * Get queue configuration
+   * @param name queue name
+   * @returns queue configuration.
+   */
+  async getQueueConfiguration(name: QueueNames): Promise<Bull.JobOptions> {
+    const queues = await this.getAllQueueConfigurations();
+    const [queueConfig] = queues.filter((queue) => queue.queueName === name);
```

Comment:
If the expectation is one and always one config instead of using `queues.filter`, `queues.find` can also be used.